packages/api-client/src/components/ImportCollection/ImportCollectionModal.vue

```diff
@@ -126,34 +162,49 @@ watch(
           class="inline-flex flex-col gap-2 items-center mt-2">
           <!-- <OpenAppButton :source="source" /> -->
           <ImportNowButton
-            :source="source"
+            :source="prefetchResult?.url ?? source"
             variant="button"
+            :watchMode="watchMode"
             @importFinished="() => $emit('importFinished')" />
         </div>
         <!-- Select the workspace -->
-        <div class="flex justify-center">
-          <div class="inline-block py-1 px-4">
-            <WorkspaceSelector />
+        <template v-if="version">
+          <div class="flex justify-center">
+            <div class="inline-flex py-1 px-4 items-center text-sm">
+              Import to: <WorkspaceSelector />
+            </div>
           </div>
-        </div>
+        </template>
+        <!-- Watch Mode -->
+        <template v-if="prefetchResult?.url">
+          <WatchModeToggle
+            v-model="watchMode"
+            :disableToolTip="true" />
+          <div>hi hello</div>
```

Comment:
Should probably remove this before merging

Reply:
this is a WIP will push something up in next hour or two :)